metacat-common-server/src/main/java/com/netflix/metacat/common/server/events/MetacatApplicationEventMulticaster.java

```diff
@@ -54,27 +54,23 @@ public MetacatApplicationEventMulticaster(
     }
 
     /**
-     * Post event asynchronously.
+     * Post event. Events will be handles synchronously or asynchronously based on the listener annotation.
      *
      * @param event event
      */
-    public void postAsync(final ApplicationEvent event) {
-        this.asyncEventMulticaster.multicastEvent(event);
-    }
-
-    /**
-     * Post event synchronously.
-     *
-     * @param event event
-     */
-    public void postSync(final ApplicationEvent event) {
+    public void post(final ApplicationEvent event) {
         super.multicastEvent(event);
```

Comment:
This method should named as postAsync if that what it does.

Reply:
Same comment as above.